lib/logmsg/logmsg.h

```diff
@@ -478,6 +476,19 @@ void log_msg_tags_foreach(const LogMessage *self, LogMessageTagsForeachFunc call
 void log_msg_format_tags(const LogMessage *self, GString *result);
 void log_msg_format_matches(const LogMessage *self, GString *result);
 
+
+static inline void
+log_msg_set_recvd_rawmsg_size(LogMessage *self, guint32 size)
+{
+  self->recvd_rawmsg_size = size;
+}
```

Comment:
As we talked about it IRL, we could make it, so the Source updates the input_event_bytes_total counter based on this field, so the descendant classes only need to call this setter, and not the stats update fn.

Reply:
Fixed.